### PR TITLE
[FIX] My webbings, where are they?

### PR DESCRIPTION
**Pull request by Spaghetti-bit**
## What Does This PR Do

Adds a new special layer for certain belts, allowing them  to go over the suit.

## Why It's Good For The Game

Bandolier, judo belt, and webbing, among other things were going under armor. Who wears a set of webbings under their armor?

## Images of changes

After:
image

After 2:
image

image
image
image
image

## Testing

Yes, loaded a local server, cried with joy.

## Changelog
:cl:
fix: Explorer Webbings now go over suit items.
fix: Captain's rapier now goes over suit items.
fix: Chef's Apron now goes over suit items.
fix: Wrestling Belt now goes over suit items.
fix: Compact Defibs now go over suit items.
fix: Null rods now go over suit items.
fix: Webbings now go over suit items.
fix: Bandoliers now go over suit items.
fix: Judo Belt now goes over suit items.
fix: Some layering issues.
/:cl:

### CODE REVIEW

**Review by DGamerL — CHANGES_REQUESTED** [review]
Please make a new layer for things that go over armor, instead of just reverting the PR

**Comment by Spaghetti-bit** [comment]
All belts that should be over suits are not over suits, if there are any I missed, I'll add them later. As for now? Most of the common sense stuff is good enough.

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -18,6 +18,9 @@
 	/// Ignores update_weight() if TRUE
 	var/large = FALSE
 
+	/// Belt goes over all suit slots if TRUE
+	var/special = FALSE
```
Comment (Sirryan2002) [inline code/game/objects/items/weapons/storage/belt.dm:22]: this should be named better, `special` gives no indication as to what the var actually does. Something like `layer_over_suit` would be a better name

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (Spaghetti-bit) [reply code/game/objects/items/weapons/storage/belt.dm]: Done.

**Review by DGamerL — APPROVED** [review]

**Review by Sirryan2002 — APPROVED** [review]

**Pull request merged**